components/backup_text/src/mask.rs

```diff
@@ -122,11 +124,12 @@ pub fn workload_sim_mask(mut datum: Datum) -> MaskResult {
         Datum::Time(time) => *time = mask_time(*time),
 
         Datum::Dec(_) | Datum::Json(_) | Datum::Set(_) => {
-            return Err(datum); // todo: not supported yet
+            // todo: not supported yet
+            warn!("mask not supported yet"; "datum" => ?datum);
```

Comment:
This log will print for each of these datums, and it can be a huge number of times depending on the user data.

Reply:
Removed 😢